src/ResourceManager/Network/Commands.Network/help/Add-AzureRmExpressRouteCircuitConnectionConfig.md

```diff
@@ -57,7 +57,7 @@ Get-AzureRmExpressRouteCircuit -Name $initiatingCircuitName -ResourceGroupName $
 A minimum /29 customer address space to create VxLan tunnels between Express Route Circuits
 
 ```yaml
-Type: System.String
```

Comment:
Please rereun the Update-MarkdownHelpModule command with -UseFullNameType parameter to get rid of all these changes - you'll need to run it on the whole module rather than just using Update-MarkdownHelp because it needs to be fixed everywhere.

Reply:
Please also be sure to rebuild and import your module into a fresh session because there's a lot of changes that I don't think are intentional.  Be sure to remove the AzureRM.Network module from Program Files/WindowsPowerShell/Modules because that sometimes gets imported and used in place of your local module.

Reply:
Sure, I will do that. Thanks for the pointers.